bndtools.core/src/bndtools/editor/common/BndEditorPart.java

```diff
@@ -17,6 +17,7 @@
 
     protected BndEditModel model;
 
+    private boolean committing = false;
```

Comment:
Does this need to be `volatile` or `AtomicBoolean`? Not sure if `commit` and `propertyChange` are called on the same thread.